src/home/room_screen.rs

```diff
@@ -2641,6 +2631,13 @@ pub enum TimelineUpdate {
     /// though the success or failure of the request is not yet known until the client
     /// requests the member info via a timeline event's `sender_profile()` method.
     RoomMembersFetched,
```

Comment:
since we changed the name of the request to use the word "Sync", we should also use that here.

```suggestion
    RoomMembersSynced,
```

Reply:
(and update the docs accordingly)